File: src/main/java/dev/macrohq/swiftslayer/util/Logger.java
package dev.macrohq.swiftslayer.util;

import net.minecraft.util.ChatComponentText;

public class Logger {
    public static void info(String message) {
        send("a" + message);
    }

    public static void error(String message) {
        send("c" + message);
    }

    private static void send(String message) {
        Ref.player().addChatMessage(new ChatComponentText("§" + message));
    }
}

File: src/main/kotlin/dev/macrohq/swiftslayer/util/Logger.kt
package dev.macrohq.swiftslayer.util

import net.minecraft.util.ChatComponentText

object Logger {
    fun info(message: Any) {
        send("a$message")
    }

    fun error(message: Any) {
        send("c$message")
    }

    fun log(message: Any) {
        send("7$message", true)
    }

    private fun send(message: String, debug: Boolean = false) {
        if (debug && !config.debugMode) return
        player.addChatMessage(ChatComponentText("§aSwift§bSlayer §8» §$message"))
    }
}
